src/util/options.schema.json

```diff
@@ -1053,14 +1053,6 @@
                 "Destabilize nodes in changed functions reluctantly",
               "type": "boolean",
               "default": false
-            },
-            "compare": {
-              "title": "incremental.reluctant.compare",
-              "description":
-                "In order to reuse the function's old abstract value the new abstract value must be leq (focus on efficiency) or equal (focus on precision) compared to the old.",
-              "type": "string",
-              "enum": ["leq", "equal"],
-              "default": "equal"
```

Comment:
Just wondering, what was the reason for removing this option? Looks like the `equal` behavior is kept, but would `leq` be incompatible with these fixes?

Reply:
Switching to `leq` would now require some hacking into the solver, as it is the solver itself that causes destabilization also beyond the reluctantly analyzed function. As there was no noticable benefit with this option, we chose not to further complicate the solver and remove the option instead.